### DISCUSSION TITLE
Ramp-up and ramp-down models

### DISCUSSION THREAD

**Discussion by jonmaddock** [ukaea/PROCESS#3315] [General]
In GitLab by @mkovari on Jan 2, 2019, 09:29

At the moment PROCESS does not have any models for the ramp-up and ramp-down.  A number of issues will arise - for example:
>  additional power (even more than what is needed for flat-top) is needed during the ramp-up phase to heat the plasma and access the H-mode. Additional heating power is needed also in the ramp-down phase to compensate for high radiation power losses. ...the heating and current drive systems are optimized to work during flat-top, resulting in possible low coupling of additional power to the plasma. A clear example is the NBI system, which suffers of shine-through losses (i.e. the part of the beam not ionized in the plasma and colliding with the first wall) at low plasma density during ramp-up and ramp-down phases.

([Estimate of 3D power wall loads due to Neutral Beam Injection in EU DEMO ramp-up phase](https://www.sciencedirect.com/science/article/pii/S2352179118300991))

 Thought would be required to determine which aspects of the ramp-up and ramp-down should be included.

**Comment by mkovari** [comment]
This will have consequences for 

- Capital cost of heating and current drive, and
- Energy consumption of heating and current drive for pulsed reactors.

I will convert this to a discussion.  Comments @ajpearcey @jmorris-uk @stuartmuldrew @jwscook @kj5248 @chris-ashe ?
